ui/src/components/ripple-out/RippleOutStoryContainer.js

```diff
@@ -135,22 +143,26 @@ const RippleOutStoryContainer = () => {
     <>
       <RippleOutHeroSection />
       <Flex
-        minH="100%"
         flexDir="column"
-        bgGradient="linear(120deg, rgba(132,225,255,0.5), rgba(197,230,212,0.5), rgba(255,235,174,0.5));"
+        background="linear-gradient(180deg, #F0F5FF 10.74%, #E9F6FF 22.35%, #FFFAEC 99.87%)"
```

Comment:
this change seems to make the gradient very faint and not aligned to the design

<img width="1469" alt="image" src="https://github.com/hmcc-global/hmcchk-web/assets/16095743/1c09b299-d2b6-48e9-a683-f236c6243f08">

Reply:
oh, should i revert it back to the original? i changed it to standardize the background across the pages so...

Reply:
technically, you are correct already and following the design. the other pages should also follow their own individual designs for the pages. so you can revert back to original

Reply:
okk, noted.